reporter/datadog_reporter.go

```diff
@@ -585,6 +643,15 @@ func addTraceLabels(labels map[string][]string, i traceFramesCounts) {
 		// This is also consistent with ddprof.
 		labels["thread id"] = append(labels["thread id"], fmt.Sprintf("%d", i.tid))
 	}
+
+	if ts != 0 { // 0 means no timeline active
+		labels["end_timestamp_ns"] =
+			append(labels["end_timestamp_ns"], strconv.FormatUint(ts, 10))
+	}
+
+	if i.comm != "" {
```

Comment:
redundant though helps with grouping, I should try to make this work without duplicating this information